packages/a11y/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "a11y",
```

Comment:
If we plan to publish these packages I think we need to have the same prefix for all of them.`ui-autotools-a11y`?